### DISCUSSION TITLE
有没有什么办法让机器人只发图片不对图片进行说明

### DISCUSSION THREAD

**Discussion by yjGjin** [Samarium150/mirai-console-lolicon#49] [Ideas]
QQ群里人比较多，有时候发图进行说明直接刷屏了

**Comment by Samarium150** [comment]
> QQ群里人比较多，有时候发图进行说明直接刷屏了

下个版本吧，我写个issue

**Comment by Samarium150** [comment]
> QQ群里人比较多，有时候发图进行说明直接刷屏了

已在`v4.1.0`实现，配置文件里`verbose: false`就没有图片信息了

**Reply by yjGjin** [reply]
感谢大佬
